js/src/common/helpers/debounce.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Ensures that the provided function isn't called multiple times in succession.
+ *
+ * Forked from BMO's work in Genesis Blocks.
+ *
+ * @param {() => any} func
+ * @param {number} wait
+ *
+ * @return {() => void} A debounced function.
+ */
+const debounce = ( func, wait ) => {
```

Comment:
It doesn't follow the same standard as the rest of this plugin, but it is possible to name the export instead of using `default`.

```
export const debounce = () => { ... }
```

That will prevent the need for `export default debounce` at the bottom, and it can be included with

```
include { debounce }  from './debounce';
```

Reply:
Yeah, this plugin's setup is a bit outdated, with the `index.js` file having all of the exports.

It might be strange to not have this `debounce` in `index.js`, while all of the other helper functions are in it.